src/Kunstmaan/MediaBundle/Validator/Constraints/IsExtensionAllowedValidator.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Kunstmaan\MediaBundle\Validator\Constraints;
+
+use Kunstmaan\MediaBundle\Helper\MediaManager;
+use Symfony\Component\Form\Exception\UnexpectedTypeException;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+
+class IsExtensionAllowedValidator extends ConstraintValidator
+{
+    /**
+     * @var MediaManager
+     */
```

Comment:
```suggestion
    /** @var MediaManager */
```

To keep the same style as the next docblock